plugins/auth-backend/api-report.md

```diff
@@ -216,9 +223,6 @@ export type GcpIapResult = GcpIapResult_2;
 // @public @deprecated
 export type GcpIapTokenInfo = GcpIapTokenInfo_2;
 
-// @public
-export function getDefaultOwnershipEntityRefs(entity: Entity): string[];
```

Comment:
Let's not remove this yet, fair number of people will be using it still and we don't want to ship an immediate breakage here. I do wonder if this might still make sense to keep around though? Although probably in `auth-node` in that case, I'm thinking for the case where you want the default but perhaps filter or add to the list a bit? Otherwise we can mark it as deprecated